### PR TITLE
Integrate login

### PR DESCRIPTION
**Pull request by ThomasHoooo**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Summary of PR

Integrated login, register with the backend
Removed `package-lock.json` because it is not needed for yarn, yarn should be used for package management instead of npm.

### CODE REVIEW

File: frontend/app/components/modal/signupModal.tsx [hunk 1]
```diff
@@ -77,7 +79,7 @@ const SignupModal: React.FC<SignUpProps> = ({ isNav = false }) => {
                 placeholder="Enter your email"
                 labelPlacement="outside"
                 errorMessage={errors.email?.message as string}
-              />
+              /> */}
```
Comment (rexong) [inline frontend/app/components/modal/signupModal.tsx:82]: still need this comment out portion?

File: frontend/app/components/modal/loginModal.tsx [hunk 2]
```diff
@@ -23,7 +29,15 @@ const LoginModal = () => {
 
   const onSubmit = handleSubmit(async data => {
     // eslint-disable-next-line no-console
-    console.log(data); //add in your api call to check login
+    const response = await POST('/login', data);
+    console.log(response); //add in your api call to check login
```
Comment (rexong) [inline frontend/app/components/modal/loginModal.tsx:33]: can remove this console.log

**Review by rexong — CHANGES_REQUESTED** [review]

File: frontend/app/components/modal/loginModal.tsx (hunk 2 above)
Comment (ThomasHoooo) [reply frontend/app/components/modal/loginModal.tsx:33]: good catch, thanks!

File: frontend/app/axios/axios.tsx [hunk 3]
```diff
@@ -8,6 +8,38 @@ const axiosInstance = axios.create({
...
+  try {
+    const response = await axiosInstance.delete(url);
+    return response;
+  } catch (error) {
+    return ErrorResponse;
+  }
+}
+
 export const getData = async (url : string) => {
   try {
```
Comment (Ugholaf) [inline frontend/app/axios/axios.tsx:44]: is this repeated? Actually, yours is better, more general. If is the same and I can remove it in my PR

**Review by Ugholaf — APPROVED** [review]
Just remove those console logs

**Pull request merged**